From 557eae5d1d45bb49f190613cf7fbb9f0c0230df1 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Mon, 8 Feb 2021 06:07:47 -0800
Subject: [PATCH 1247/1921] arch/arm64: dts: fix the i2c GPIO expander for
 CN9130-CRB

The CN9130-CRB has i2c expander on bus 0, address 0x20.
The board schematics release R1P2 is wrongly states it on 0x27.

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: Ibfe03e9f881bcd5d3af9941594d91c44f8e63fb1
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/45552
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-crb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-crb.dts b/arch/arm64/boot/dts/marvell/cn9130-crb.dts
index 97f56316356b..3d50b97fe126 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-crb.dts
+++ b/arch/arm64/boot/dts/marvell/cn9130-crb.dts
@@ -139,7 +139,7 @@
 		pinctrl-names = "default";
 		gpio-controller;
 		#gpio-cells = <2>;
-		reg = <0x27>;
+		reg = <0x20>;
 		status = "okay";
 	};
 };
-- 
2.31.1

